### DISCUSSION TITLE
Welcome to apkshellext Discussions!

### DISCUSSION THREAD

**Discussion by kkguo** [kkguo/apkshellext#108] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blackcrack** [comment]
hi,

sorry, but it's funny, by the ""whole" work" what you did in the last month to developing further this nice program...
and now only discussion without any code in the last couple month ?

there be enough issues who should make before..

**Comment by kkguo** [comment]
@blackcrack , that's not the whole work for sure, just not ready for a new release. I'm rewriting (again) a lot of code with C++. Just because C# has plenty of restriction and brings a lot of issues, and no solution from the root.
Regarding this board, I just find this thing yesterday while I was surfing github. And I think it's interesting. feel free to talk anything here.

**Reply by blackcrack** [reply]
so, i looking forward and be curious when there is a new version get, because i love apkshellext very and wait for the new support of the new extensions like apkx, apks and so far.. ... and by the way, sorry be not disappointed, but i want ask, in which land you lives.. because the issue of Covit and be able to imagine for yourself what is approximate .. as example in Germany becomes we really good supported with in informations about worldwide situations what's at moment the state of the stand is .. because that, your land ?

best
